src/main/scala/li/cil/oc/common/tileentity/Relay.scala

```diff
@@ -232,6 +237,7 @@ class Relay extends traits.SwitchLike with traits.ComponentInventory with traits
       case driver if driver.slot(stack) == Slot.HDD => maxQueueSize = queueBaseSize
       case driver if driver.slot(stack) == Slot.Card =>
         isWirelessEnabled = false
+        wirelessTier = Tier.One
```

Comment:
I don't think it adds any benefit to change this value - it is a bit misleading because if the "is enabled" booleans are respected than this value has no effect

Reply:
I have to set this in order to have the relay respect the maximum signal strength for each card. If anything, it should replace isWirelessEnabled and should be set to -1 to mean no wireless.